sdk/internal/recording/recordings/TestBodyKeySanitizer.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Entries": [],
+  "Variables": {}
+}
```

Comment:
Is this a bug that the recordings are emitted without a final newline?

Reply:
What is the importance of having a final newline in the recordings?

Reply:
@RickWinter they are just serialized json. I will take action to add a trailing newline if one doesn't exist already 👍